content/source/docs/cloud/getting-started/policies.html.md

```diff
@@ -1,24 +1,24 @@
 ---
 layout: "cloud"
-page_title: "Configuring Sentinel Policies - Getting Started - Terraform Cloud"
+page_title: "Configuring Policies - Getting Started - Terraform Cloud"
 ---
 
-# Configuring Sentinel Policies
+# Configuring Policies
 
 **Prerequisites:** Before starting this guide, make sure you've [configured VCS access](./vcs.html) and [successfully completed a run](./runs.html).
 
-## About Sentinel Policies in Terraform Cloud
+## About Policies in Terraform Cloud
 
-[Sentinel](https://www.hashicorp.com/sentinel) is an embedded policy-as-code framework integrated with various HashiCorp products. It enables fine-grained, logic-based policy decisions, and can be extended to use information from external sources. Terraform Cloud enables users to enforce Sentinel policies during runs.
+[Sentinel](https://www.hashicorp.com/sentinel) is an embedded policy-as-code framework integrated with various HashiCorp products. It enables fine-grained, logic-based policy decisions, and can be extended to use information from external sources. Terraform Cloud enables users to enforce policies during runs.
 
 A policy consists of:
 
-- The Sentinel policy code
+- The policy controls defined as code
 - An enforcement level that changes how a policy affects the run lifecycle
 
 ### Enforcement Levels
 
-Enforcement levels in Sentinel are used for defining behavior when policies fail to evaluate successfully. Sentinel provides three enforcement modes:
+Enforcement levels in Terraform are used for defining behavior when policies fail to evaluate successfully. Terraform provides three enforcement modes:
```

Comment:
This is a place where we don't want to switch. Enforcement is purely a Sentinel concept. We still introduce Sentinel in this guide so I think it is fine to introduce Sentinel concepts throughout.

Reply:
I've rolled this back